src/services/PostCache/cachePosts.ts

```diff
@@ -0,0 +1,23 @@
+import type { InterfacePost } from "../../models";
+import PostCache from "../redisCache";
+
+// Function to store posts in the cache using pipelining
+export async function cachePosts(posts: InterfacePost[]): Promise<void> {
+  try {
+    const pipeline = PostCache.pipeline();
+
+    posts.forEach((post) => {
+      if (post !== null) {
+        const key = `post:${post._id}`;
+        pipeline.set(key, JSON.stringify(post));
+        // SET the time to live for each of the organization in the cache to 300s.
+        pipeline.expire(key, 300);
+      }
+    });
+
+    // Execute the pipeline
+    await pipeline.exec();
+  } catch (error) {
+    console.log(error);
```

Comment:
Do we use console log for all our logging. If so then this is fine for now but we should look into a new logging tool such as Bunyan.

Reply:
Yeah Sorry, that's actually my bad, WIll make changes to remove console.log. 

I think we do use a logger library but I think that is more for logging errors for now.